Added all files and README

theAdityaNVS opened the pull request:
## Pull Request Checklist

Go through the check boxes given below and make sure you mark the relevant ones.
#### To check put x inside [ ] eg. [x] Extension

Is your project an
- [✔️ ] Extension 
- [ ] Application


Given below are the mandatory requirements we expect from your project. Your Pull Request will be reviewed only if all the minimum requirements relevant to your project are satisfied.

- [✔️ ] Added readme.md to my project folder with relevant information.
- [✔️ ]  I have provided the demo video link (showing the functionality in action) or the public deployment link of my extension of application within the readme.md (if Applicable).
- [✔️ ] I have explained the process in a readable manner within readme.md.

### Terms and Conditions

`Plagiarism check from Team Crio will be solid and if your code is found to be plagiarized, the team has every right to reject your Pull Request and take necessary actions.`
- [✔️ ] I can assure you that this is my own contribution and I did not involve in any kind of plagiarism activities.

Buckle up and wait till you receive communication from the review team. Keep a tab on this pull request page for comments on your project. Any necessary suggestions will also be provide here. All the best :)

#### Once your PR get's merged, take some time ⏲️, celebrate 🥳 and share it with the community on slack.

theAdityaNVS committed:
Added all files and README

sudhanshutiwari264 commented:
@theAdityaNVS you've done a great job :clinking_glasses: it matches the structure I had planned :+1: 
@archithdwij could you please review this!
Let's wait until he reviews! 
Celebrate! Take a moment to relax!

Nitesh-thapliyal commented on Meet Up - JavaScript/README.md:
```diff
@@ -0,0 +1,15 @@
+# chrome-extension-meetup
+Its a simple extension which can open and store all the added meeting links to join quickly.
```
There are some grammatical mistakes present, please correct them.

theAdityaNVS replied:
Yes I have updated the README

Nitesh-thapliyal commented:
Good work @theAdityaNVS 

Here is a suggestion that you can work on :

- Buttons are overlapping the Links, can you make that right?

![Screenshot (1158)](https://user-images.githubusercontent.com/53345517/124398847-8936b080-dd35-11eb-8396-d73f34aec3ee.png)

Nitesh-thapliyal commented:
Hey Aditya, 

I just found a bug in your extension:

- Rename feature is not working instead it is creating a new link and extra tabs.
For better understanding I have created a video, go through the video and try to fix it asap
 


https://user-images.githubusercontent.com/53345517/124399139-2d6d2700-dd37-11eb-8d85-5bc909c6c6c1.mp4

Nitesh-thapliyal reviewed (changes requested):
Hey Aditya,

I just found a bug in your extension:

- Rename feature is not working instead it is creating a new link with extra tabs.

For better understanding I have created a video, go through the video and try to fix it asap

https://user-images.githubusercontent.com/53345517/124399330-432f1c00-dd38-11eb-9f24-ccaa7f76ad8f.mp4

theAdityaNVS committed:
updated README

theAdityaNVS committed:
fixed rename function

theAdityaNVS committed:
fixed icons and rename function

theAdityaNVS commented:
Hi,
- So the empty list error is fixed now.
- It also seems like Heroku isn't allowing few functions so I deployed it on GitHub Pages [Here](https://theadityanvs.github.io/chrome-extension-meetup/).
- Also I have fixed padding issues as bootstrap CDN broke and i fixed it.
- I have also updated README and added screenshot.

Nitesh-thapliyal reviewed (approved):
LGTM

Nitesh-thapliyal commented on Meet Up - JavaScript/README.md:
```diff
@@ -1,15 +1,29 @@
-# chrome-extension-meetup
-Its a simple extension which can open and store all the added meeting links to join quickly.
+# Chrome-Extension-MeetUp
+Its a simple extension which store all your meeting links together in one place and in one click you can join quickly.
```
Still, there is grammatical mistake use  Grammarly to avoid grammatical mistakes

archithdwij commented:
Hi @theAdityaNVS I like the idea of managing the meeting links in one place. Your work looks solid :) 
I have a small suggestion and that's because I am just curious. 
Is it possible to link the meetings scheduled in the calendar/Gmail and update the extension automatically? Just a thought. Do it as a future upgrade or something if it's possible. Other than that, congratulations on getting your PR merged 💯

jainaayush01 reviewed (changes requested):
Please remove the ".DS_Store" files from the Pull Request. 
Just add it to .gitignore, the file has no relevance on other computers.

theAdityaNVS committed:
removed other files

theAdityaNVS commented:
Yes @archithdwij  , Will definitely try to implement and yes thats possible through calendar integration. 
But to implement in extension it's a bit complicated due to chrome content security policies. 

Also the PR didn't get merged.

theAdityaNVS commented:
@jainaayush01 
I have removed those files.

archithdwij commented:
I saw aayush's suggestions after my review so I waited for you to remove the .DS_Store file. Will get it merged now :)

archithdwij commented:
@all-contributors Please add @theAdityaNVS for code, design, idea
